src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Controllers/PublicationsController.cs

```diff
@@ -40,4 +40,23 @@ public PublicationsController(IPublicationService publicationService)
                 search: request.Search)
             .HandleFailuresOrOk();
     }
+
+    /// <summary>
+    /// Get a publication’s details"
```

Comment:
- Random double quotes at the end of this
- Also, noticed that the summary for `ListPublications` should be `List publications` (lowercased 'publications')

Reply:
done :)